[0m20:10:27.480520 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7dbb92b0cb90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7dbb92ed4fb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7dbb92aa6240>]}


============================== 20:10:27.483572 | d8c0f1da-b77f-4ee8-8107-5db571727f6b ==============================
[0m20:10:27.483572 [info ] [MainThread]: Running with dbt=1.10.6
[0m20:10:27.484344 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/codespace/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init src', 'send_anonymous_usage_stats': 'True'}
[0m20:10:27.493600 [debug] [MainThread]: Starter project path: /workspaces/POC-Data-Engineering/dbt/.venv/lib/python3.12/site-packages/dbt/include/starter_project
[0m20:10:27.496250 [info ] [MainThread]: 
Your new dbt project "src" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:10:27.496915 [info ] [MainThread]: Setting up your profile.
[0m20:10:30.756618 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 3.344871, "process_in_blocks": "0", "process_kernel_time": 0.122734, "process_mem_max_rss": "112872", "process_out_blocks": "80", "process_user_time": 1.412939}
[0m20:10:30.757399 [debug] [MainThread]: Command `dbt init` succeeded at 20:10:30.757236 after 3.35 seconds
[0m20:10:30.757996 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7dbb92e652b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7dbb927bc1d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7dbb91d66b10>]}
[0m20:10:30.758535 [debug] [MainThread]: Flushing usage events
[0m20:10:31.103663 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m20:15:10.144520 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7943393c9e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x794338fbc380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7943393c9a00>]}


============================== 20:15:10.149361 | 152ab15f-6a31-4f44-a7e6-8f7eb9e684ef ==============================
[0m20:15:10.149361 [info ] [MainThread]: Running with dbt=1.10.6
[0m20:15:10.153089 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/home/codespace/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt run', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True'}
[0m20:15:10.157437 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/POC-Data-Engineering/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:15:10.159317 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.14833644, "process_in_blocks": "2368", "process_kernel_time": 0.117386, "process_mem_max_rss": "98860", "process_out_blocks": "16", "process_user_time": 1.50038}
[0m20:15:10.161547 [debug] [MainThread]: Command `dbt run` failed at 20:15:10.161370 after 0.15 seconds
[0m20:15:10.163990 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x794339517ef0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7943390b7dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79433966f6e0>]}
[0m20:15:10.164877 [debug] [MainThread]: Flushing usage events
[0m20:15:10.506891 [debug] [MainThread]: An error was encountered while trying to flush usage events
